src/services/consent/hooks/consentversions.js

```diff
@@ -1,7 +1,13 @@
+const auth = require('@feathersjs/authentication');
+// const globalHooks = require('../../../hooks');
+
+// todo check userId
 exports.before = {
-	all: [],
-	find: [],
-	get: [],
+	all: [auth.hooks.authenticate('jwt')],
```

Comment:
did you think of the registration, making sure it always uses the most recent version?
Without having put much thought into it, is it possible unregistered users have to access the consent version?